src/dmd/backend/cod1.d

```diff
@@ -3117,6 +3153,108 @@ int FuncParamRegs_alloc(ref FuncParamRegs fpr, type* t, tym_t ty, reg_t* preg1,
     return 1;
 }
 
+/***************************************
+ * Finds replacemnt types for register passing of aggregates.
+ */
+void argtypes(type *t, type **arg1type, type **arg2type)
```

Comment:
can `arg1type` and `arg2type` be `ref type*` instead? 
Also please write `type* arg` not `type *arg`, as this is not C.

Reply:
OK.